src/bosh-google-cpi/integration/stemcell_test.go

```diff
@@ -29,12 +53,24 @@ var _ = Describe("Stemcell", func() {
 			  "method": "delete_stemcell",
 			  "arguments": ["%v"]
 			}`, stemcellCID)
+		assertSucceeds(request)
+	})
 
-		response, err := execCPI(request)
-		Expect(err).ToNot(HaveOccurred())
-		Expect(response.Error).To(BeNil())
-		Expect(response.Result).To(BeNil())
-
+	//This currently fails due to issue: https://github.com/google/google-api-go-client/issues/181
```

Comment:
Does the sha1 serve a purpose if it's not validated on disk creation? If not, then I'll find out more about when/if the validation will be implemented.